unit-tests/src/test/scala/scala/scalanative/unsafe/CStringSuite.scala

```diff
@@ -16,9 +16,6 @@ object CStringSuite extends tests.Suite {
     fromCString(c"\a"); fromCString(c"\\a")
     fromCString(c"\b"); fromCString(c"\\b")
     fromCString(c"\f"); fromCString(c"\\f")
-    fromCString(c"\n"); fromCString(c"\\n")
-    fromCString(c"\r"); fromCString(c"\\r")
-    fromCString(c"\t"); fromCString(c"\\t")
```

Comment:
It seems to me that we should move *all* tests from this `test(...)` to the one below, with proper `assertEquals` for all of them. There's no point not to check the actual values for some of the strings.